tests/e2e/test_pgsql_pod_respin.py

```diff
@@ -0,0 +1,146 @@
+"""
+Module to perform PGSQL workload
+"""
+import logging
+import pytest
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.utility import templating, utils
+from ocs_ci.utility.utils import run_cmd, TimeoutSampler
+from ocs_ci.ocs.utils import get_pod_name_by_pattern
+from ocs_ci.ocs.ripsaw import RipSaw
+from ocs_ci.ocs import constants
+from tests import disruption_helpers
+from ocs_ci.framework.testlib import E2ETest, google_api_required, workloads
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.utility.spreadsheet.spreadsheet_api import GoogleSpreadSheetAPI
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope='function')
+def ripsaw(request, storageclass_factory):
+
+    # Create storage class
+    log.info("Creating a Storage Class")
+    storageclass_factory(sc_name='pgsql-workload')
+
+    # Create RipSaw Operator
+    ripsaw = RipSaw()
+
+    def teardown():
+        ripsaw.cleanup()
```

Comment:
Are we reverting back to the default namespace in the cleanup?

Reply:
The other PR will fix this in the class method, so cleanup() will revert the namespace.

Reply:
Reverting to default namespace was fixed in PR  746.